dataquality/core/init.py

```diff
@@ -21,43 +21,45 @@
 BAD_CHARS_REGEX = r"[^\w -]+"
 
 
-class _Init:
-    def get_project_by_name_for_user(self, project_name: str) -> Dict:
-        return api_client.get_project_by_name(project_name)
+class InitManager:
+    def get_or_create_project(
```

Comment:
let's keep your clean layout, but still capture that duplicate project name error by retrying this function once on a GalileoException

```suggestion
    @retry(stop=stop_after_attempt(1), wait=wait_fixed(0.5), retry_if_exception_type(GalileoException))
    def get_or_create_project(
```